client/tests/Thumbnail.test.js

```diff
@@ -18,4 +19,12 @@ describe('Unit Test for <Thumbnail /> Component', () => {
     const component = (<Thumbnail photoId={15} />);
     expect(component.props.photoId).toEqual(15);
   });
+
+  it('should call makeCurrentSlide on click of thumbnail', () => {
+    const makeCurrentSlide = sinon.spy();
```

Comment:
If you would like to limit your dependencies you can also monitor whether a function has been called using a mock function in Jest: https://facebook.github.io/jest/docs/en/mock-functions.html